test/command.test.ts

```diff
@@ -381,6 +382,70 @@ describe("Command", () => {
                 });
             });
         });
+
+        describe("ignoreCase", () => {
+            describe("true", () => {
+                it("should match a command in a case-insensitive manner", () => {
+                    m.text = "/START";
+                    m.entities = [{
+                        type: "bot_command",
+                        offset: 0,
+                        length: 6,
+                    }];
+                    const ctx = new Context(update, api, me);
+                    assert(
+                        Command.hasCommand("start", {
+                            ...options,
+                            ignoreCase: true,
+                        })(ctx),
+                    );
+
+                    m.text = "/start";
+                    m.entities = [{
+                        type: "bot_command",
+                        offset: 0,
+                        length: 6,
+                    }];
+                    assert(
+                        Command.hasCommand("start", {
+                            ...options,
+                            ignoreCase: true,
+                        })(ctx),
+                    );
+                });
+            });
+
+            describe("false", () => {
+                it("should match a command in a case-sensitive manner", () => {
+                    m.text = "/START";
+                    m.entities = [{
+                        type: "bot_command",
+                        offset: 0,
+                        length: 6,
+                    }];
+                    const ctx = new Context(update, api, me);
+                    assertFalse(
+                        Command.hasCommand("start", {
+                            ...options,
+                            ignoreCase: false,
+                        })(ctx),
+                    );
+
+                    m.text = "/start";
+                    m.entities = [{
+                        type: "bot_command",
+                        offset: 0,
+                        length: 6,
+                    }];
+                    assert(
+                        Command.hasCommand("start", {
+                            ...options,
+                            ignoreCase: false,
+                        })(ctx),
+                    );
+                });
+            });
+        });
     });
```

Comment:
There are no test cases for regular expressions. Should they be added?

Reply:
Yes. I'll do that soon ™️

Reply:
I'm on it